pkg/odo/cli/component/common_push.go

```diff
@@ -183,6 +183,10 @@ func (cpo *CommonPushOptions) Push() (err error) {
 	cmpName := cpo.LocalConfigInfo.GetName()
 	appName := cpo.LocalConfigInfo.GetApplication()
 	cpo.sourceType = cpo.LocalConfigInfo.GetSourceType()
+	forceWrite := false
+
+	resolvePath := ""
+	var newFileMap map[string]util.FileData
```

Comment:
Can we please add comment for the three variables introduced in this and `adapter.go` file?